app/assets/stylesheets/application.scss

```diff
@@ -27,6 +27,19 @@
 a:hover {
   text-decoration: underline;
 }
+
+footer, nav {
+  span.spacer{
```

Comment:
Avoid qualifying class selectors with an element.<br>Opening curly brace `{` should be preceded by one space